diff --git a/examples/simpletest.py b/examples/simpletest.py
index 23f9f0c..db35596 100644
--- a/examples/simpletest.py
+++ b/examples/simpletest.py
@@ -10,16 +10,16 @@ import Adafruit_MCP3008
 
 
 # Software SPI configuration:
-CLK  = 18
-MISO = 23
-MOSI = 24
-CS   = 25
-mcp = Adafruit_MCP3008.MCP3008(clk=CLK, cs=CS, miso=MISO, mosi=MOSI)
+#CLK  = 18
+#MISO = 23
+#MOSI = 24
+#CS   = 25
+#mcp = Adafruit_MCP3008.MCP3008(clk=CLK, cs=CS, miso=MISO, mosi=MOSI)
 
 # Hardware SPI configuration:
-# SPI_PORT   = 0
-# SPI_DEVICE = 0
-# mcp = Adafruit_MCP3008.MCP3008(spi=SPI.SpiDev(SPI_PORT, SPI_DEVICE))
+SPI_PORT   = 0
+SPI_DEVICE = 0
+mcp = Adafruit_MCP3008.MCP3008(spi=SPI.SpiDev(SPI_PORT, SPI_DEVICE))
 
 
 print('Reading MCP3008 values, press Ctrl-C to quit...')
